source/common/modules/markdown-editor/plugins/render-tables.ts

```diff
@@ -86,6 +86,16 @@ function writeTableToDocument (cm: CodeMirror.Editor, table: TableEditor): boole
   // because such a table header WILL NEVER be on the last line, plus
   // this way we can check for Setext headers without having to worry.
 
+  // render links in table
```

Comment:
I would suggest to do that in the TableEditor's element itself, since the editor just searches through all those elements already, and by performing this check only when a table cell has changed this makes it both more performant and more compartmentalized. The TableEditor requires further changes anyways.

Reply:
I have implemented using suggestion.